app/views/user_message_statuses/index.html.haml

```diff
@@ -13,4 +13,4 @@
         %td
           = user_message_status.message.created_at.strftime(Settings.date_time_format)
         %td
-          = link_to 'View claim', __send__("#{current_user.persona.class.to_s.pluralize.underscore}_claim_path", user_message_status.message.claim) + "#messages"
+          = link_to 'View claim', __send__(current_user.persona.message_claim_path, user_message_status.message.claim, anchor: 'messages')
```

Comment:
Unfortunately there isn't a base class. :( They both just inherit from `ActiveRecord::Base` and I don't fancy implementing an entire class to inherit this method!